frontend/packages/shared/src/api/queries.ts

```diff
@@ -174,3 +175,39 @@ export const getProcessTaskType = (org: string, app: string, taskId: string) =>
 
 // Contact Page
 export const fetchBelongsToGiteaOrg = () => get(belongsToOrg());
+
+// Org level code lists
+
+const orgLevelCodeListsMock: CodeList[] = [
+  [
+    {
+      description: 'description1',
+      helpText: 'helpText1',
+      label: 'label1',
+      value: 'value1',
+    },
+    {
+      description: 'description2',
+      helpText: 'helpText2',
+      label: 'label2',
+      value: true,
+    },
+  ],
+  [
+    {
+      description: 'description3',
+      helpText: 'helpText3',
+      label: 'label3',
+      value: 3,
+    },
+  ],
+];
+export const getOrgLevelCodeLists = async (): Promise<CodeList[]> =>
+  // TODO: Replace with endpoint when it is ready in backend.
+  new Promise((resolve) => {
+    setTimeout(() => {
+      // Replace the two resolves to test with empty list
+      // resolve([]);
+      resolve(orgLevelCodeListsMock);
+    }, 1000);
```

Comment:
Godt tenkt å simulere treghet i responsen, men et helt sekund er kanskje litt lenge? Det kan det jo bli litt tungt å jobbe med. Har du vurdert kortere tid, typ 100-200 millisekender?

Reply:
Satt bare det tallet vi brukte på Ansattporten. Er helt samme for min del, så kan endre til 200 😄